Add output plugin for Azure Log Analytics (#5355)
influxdata/telegraf: plugins/outputs/azure_monitor_logs/README.md

# Azure Log Analytics Output Plugin

> This plugin sends metrics in a Azure Log Analytics.

Reviewer: ```suggestion
This plugin sends metrics to Azure Log Analytics.
```

## Configuration

```toml

> # A plugin that can transmit metrics over HTTP

Reviewer: I guess that's a copy'n paste leftover... ;-)

[[outputs.azure_monitor_logs]]
  ## Customer ID (Workstation ID) and Key for Azure Log Analytics resource.
  # customer_id = "<Workstation ID>"

>   # shared_key = "<Secret>"

Reviewer: Please add some documentation for the `shared_key` option.
